### PR TITLE
Change the updated images of tide.

### PR DESCRIPTION
**Pull request by Summer-Field**
# Change the updated images of tide.

## Config cluster's secret.
Steps are as follows:
1. `echo <github_token> | base64` to generate the token
2. `kubectl edit secret -n prow github-token`
3. add the following line below the `cert:`  line in yaml file:
``` yaml
token: <your-generated-token>
```

## Config cluster's tide deployment
add the following lines in env of tide depoyment:
``` yaml
- name: GITHUB_TOKEN
  valueFrom:
    secretKeyRef:
      name: github-token
      key: token
```

### CODE REVIEW

**Comment by Jeffwan** [comment]
image
Can you *carefully* look at the code I wrote..?

**Comment by Summer-Field** [comment]
Update.

File: starter-s3.yaml [hunk 1]
```diff
@@ -455,6 +455,11 @@ spec:
             secretKeyRef:
               name: github-token
               key: appid
+        - name: GITHUB_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: github-token
```
Comment (Jeffwan) [inline starter-s3.yaml]: Can you use a different token instead of editing and reusing the existing one?

**Comment by Jeffwan** [comment]
> 1. echo <github_token> | base to generate the token
> 2. kubectl edit secret -n prow github-token

These two steps don't look good to me. Let's use a new token instead of polluting existing one.
Our changes is kind of hacky and it's not a production ready change. Why do you think to editing existing one?

**Comment by Summer-Field** [comment]
Update the corresponding Doc and code.

**Comment by Jeffwan** [comment]
Looks good to me\

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**